src/proto/BUILD

```
@@ -1,36 +1,56 @@
-load("@rules_proto//proto:defs.bzl", "proto_library")
-load("//proto:nanopb.bzl", "nanopb_proto_library")
-load("@com_google_protobuf//:protobuf.bzl", "py_proto_library")
+load("@nanopb//extra/bazel:nanopb_cc_proto_library.bzl", "cc_nanopb_proto_library")
+load("@protobuf//bazel:proto_library.bzl", "proto_library")
+#load("//proto:nanopb.bzl", "nanopb_proto_library")
+
+load("@protobuf//bazel:py_proto_library.bzl", "py_proto_library")
 
 package(default_visibility = ["//visibility:public"])
 
+proto_library(
+    name = "geneva_proto",
+    srcs = [
+        "geneva_slot.proto",
+    ],
+)
+
+proto_library(
+    name = "robot_proto",
+    srcs = [
+        "geometry.proto",
+        "robot_crash_msg.proto",
+        "robot_log_msg.proto",
+        "robot_status_msg.proto",
+        "tbots_timestamp_msg.proto",
+    ],
+    deps = [
+        "geneva_proto",
```

does this work? shouldn't you need to prefix this with ":"

```suggestion
        ":geneva_proto",
```

---

for some reason, it works